zad5/app/views/artists/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <h2>Artist: <%= @artist.name %></h2>
+  <%= @artist.tracks %>
+  <%= render partial: 'common/tracks_table', locals: {tracks: @artist.tracks.pluck(:id, :title, :created_at, :updated_at).flatten} %>
```

Comment:
a czemu nie przesłać po prostu `locals: { tracks: @artist.tracks }`
fajnie wykombinowane, na ostatnim wykładzie były zagnieżdżone resource, więc będąc wzbogaconym o tę wiedzę można zrobić mniej więcej  coś takiego:
```
resources :tracks
resources :artists do
  resources :tracks, only: [:index]
end
```
to wtedy nie trzeba się o takie rzeczy martwić, bo masz tracki, które możesz sobie zescopować do kontretnego artysty :)

do trzymania wspólnych partiali polecam katalog `app/views/application`, railsy szukają tam z automatu i nie trzeba podawać pełnej ścieżki.

Reply:
Na początku tak właśnie przesyłałem, ale nie mogłem rozwiązać problemu tego nila, który jak w komenatrzu u dołu mi wyjaśniłeś okazał się trywialny i kombinowałem na różne sposoby, że może coś przy przekazywaniu jest nie tak. Dzięki wielkie za te rady!